docker-compose/pg/docker-compose.yml

```diff
@@ -26,6 +26,8 @@ services:
     restart: always
     volumes: 
       - "db_data:/var/lib/postgresql/data"
+    ports:
+      - "5432:5432"
```

Comment:
added accidentally. but since it doesn't harm specifying port number - left as is.

Reply:
Let's revert the change if not required

Reply:
I second that.

Reply:
Removed the port maping